lib/Compilation.js

```diff
@@ -838,6 +838,7 @@ Compilation.prototype.createChunkAssets = function createChunkAssets() {
 			var usedHash = useChunkHash ? chunkHash : this.fullHash;
 			if(this.cache && this.cache["c" + chunk.id] && this.cache["c" + chunk.id].hash === usedHash) {
 				source = this.cache["c" + chunk.id].source;
+				source.useCache = true;
```

Comment:
Adding this property due to performance considering. It doesn't need to invoke other function like `source()`. But is there any better way ?  @sokra